docs/HACKING.md

```diff
@@ -55,10 +55,18 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
     export PROJECT_HOME=$HOME/devel
     source /usr/local/bin/virtualenvwrapper.sh
 
+Wherever you add these flags, ensure to source it on all your open terminal windows. On most MAC OS's the ".bashrc" file does not exist in the "~/.bashrc" path. You might be able to find equivalent files like "/etc/bash_profile" or "/etc/profile". You can also create your own "/etc/bashrc" file. If you do create your own file, add these lines to your bash_profile: 
+
+    if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+    fi 
+
```

Comment:
I did not have one. All of my settings are sourced from /etc/bash_profile. Some very long stackoverflow threads exist explaining the reason why. 

Also note the words "On most MAC OS's" :)